protocol/avdl/kbfs.avdl

```diff
@@ -20,4 +37,9 @@ protocol kbfs {
         */
   void FSEvent(FSNotification event);
 
+  /**
+    TLFMetadataUpdate is called with a batch of TLFMetadata objects containing
+    new TLF metadata.
+    */
+  void TLFMetadataUpdate(array<TLFMetadataNotification> md);
```

Comment:
All of this code should be a separate file, since the current KBFS protocol is for: client=kbfsfused and service=kebase-service.

I would suggest inaugurating our feature of having protocol inclusions, and moving all of this code into a backend repo, out of the client repo.